packages/backend/src/graphql/scalars/decimal.ts

```diff
@@ -0,0 +1,13 @@
+import { Prisma } from '@progwise/timebook-prisma'
+
+import { builder } from '../builder'
+
+export const DecimalScalar = builder.scalarType('Decimal', {
```

Comment:
You exported this but didn't use it. My comment on `invoiceItem.ts` on the `hourlyRate` is related to this import.